.travis.yml

```diff
@@ -3,17 +3,34 @@ matrix:
   include:
   - python: "3.5"
   - python: "nightly"
+  - os: osx
+    language: generic
   allow_failures:
   - python: "nightly"
 
 before_install:
- - "pip install pyinstaller"
- - "pip install PyQt5"
+ - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+     wget https://github.com/ActivityWatch/activitywatch/raw/master/scripts/ci/setup_venv_osx.sh;
+     chmod +x setup_venv_osx.sh;
+     ./setup_venv_osx.sh;
```

Comment:
Holy shit, I can't believe all this shit was caused due to one missing semicolon on this line.